runtime/common/src/constants.rs

```diff
@@ -62,7 +62,7 @@ pub mod time {
 	// 1 in 4 blocks (on average, not counting collisions) will be primary BABE blocks.
 	pub const PRIMARY_PROBABILITY: (u64, u64) = (1, 4);
 
-	pub const EPOCH_DURATION_IN_BLOCKS: BlockNumber = 4 * HOURS;
+	pub const EPOCH_DURATION_IN_BLOCKS: BlockNumber = 2 * MINUTES;
```

Comment:
We cannot merge this PR into the `dev` branch. Whatever is related to specific testing needs to live in a specific separate branch and has its own CI/CD pipeline if needed